apm-sniffer/apm-sdk-plugin/jdbc-commons/src/main/java/org/apache/skywalking/apm/plugin/jdbc/PreparedStatementParameterBuilder.java

```diff
@@ -58,6 +57,11 @@ private String getParameterString() {
             }
             stringBuilder.append(parameter);
             first = false;
+            
+            //  cut the string as soon as it reached the length limitation
+            if (maxLength > 0 && stringBuilder.length() > maxLength){
+                return String.format("[%s]", stringBuilder.toString().substring(0, maxLength) + "...");
```

Comment:
Why don't you place `...` in `[%s.]` like `[%s....]`.  :-)

Reply:
> Why don't you place `...` in `[%s]` like `[%s....]`. :-)

nice